ui/pages/confirm-import-token/confirm-import-token.stories.js

```diff
@@ -11,45 +9,39 @@ import ConfirmAddToken from '.';
 export default {
   title: 'Pages/ConfirmImportToken',
   id: __filename,
+
+  argTypes: {
+    pendingTokens: {
+      control: 'object',
+      table: { category: 'Data' },
+    },
+  },
 };
 
-const history = createBrowserHistory();
+const PageSet = ({ children, pendingTokens }) => {
+  const { metamask: state } = store.getState();
 
-const PageSet = ({ children }) => {
-  const symbol = text('symbol', 'TRDT');
-  const state = store.getState();
-  const pendingTokensState = state.metamask.pendingTokens;
-  // only change the first token in the list
   useEffect(() => {
-    const pendingTokens = { ...pendingTokensState };
-    pendingTokens['0x33f90dee07c6e8b9682dd20f73e6c358b2ed0f03'].symbol = symbol;
```

Comment:
I didn't see the benefit of having this here. I think the intention of using the `text` knob was to hook into the controls, but there was no control setup for this.  The user is now about to update the entire list of `pendingTokens` with the controls added in this PR.